docs/indexer/quickstart.md

```diff
@@ -2,5 +2,158 @@
 title: Quick Start
 sidebar_label: Quick Start
 sidebar_position: 2
-description: xxx
----
\ No newline at end of file
+description: Instructions for installing and running the Shinzo Indexer
+---
+
+The **Shinzo Indexer** is the entry point into the Shinzo Network. The indexer is a client built with golang allowing a validator of any network to earn an additional reward for sorting and storing blocks with Shinzo.
+
+## Prerequisites
+
+- Make sure to have minimum Go version 1.24 or higher. You can download the latest Go verion from [here](https://go.dev/doc/install).
+- Running Ethereum node with JSON-RPC and WebSocket access (GCP Managed Blockchain Node recommended).
+
+## Installation
+
+1. Clone the repository:
+
+```bash
+git clone https://github.com/shinzonetwork/shinzo-indexer-client.git
+cd shinzo-indexer-client
+```
+
+2. Install Go dependencies:
+
+```bash
+go mod download
+```
+
+3. Once all the dependencies are installed, create a `.env` file which will have all the environment variables. You can copy and paste the below content in your `.env` file
+
+```bash
+GETH_RPC_URL=<your-geth-node-url>
+GETH_WS_URL=<your-geth-ws-url>
+GETH_API_KEY=<your-geth-api-key>        # Optional only if Geth has authentication
+
+# DefraDB Configuration
+DEFRADB_URL=                            # Keep it empty for embedded DefraDB
+DEFRADB_KEYRING_SECRET=<pick_a_password>
+
+# DEFRADB_PLAYGROUND=true
```

Comment:
suggestion: might be worth mentioning somewhere in this page how to run playground and how to access it